Solutions/CohesitySecurity/Tests/helios.py

```diff
@@ -107,38 +106,44 @@ def get_days_ago_timestamp(days_ago):
     return [jsObj["id"] for jsObj in response.json()]
 
 
-def get_recoveries(cluster_id: str, api_key: str, start_time_usecs=None, end_time_usecs=None) -> dict:
+def get_recoveries(
+    cluster_id: str, api_key: str, start_time_usecs=None, end_time_usecs=None
+) -> dict:
     """
     This function sends a GET request to the Helios API to get a list of all
     recoveries for a given Cohesity cluster.
 
     Args:
         cluster_id (str): The cluster ID of the Cohesity cluster.
         api_key (str): The API key to authenticate the request.
-        start_time_usecs (str): The start time for the recoveries (in microseconds).
-        end_time_usecs (str): The end time for the recoveries (in microseconds).
+        start_time_usecs (int, optional): The start time for the recoveries (in
+            microseconds).
+            If None, this value will not be included in the request.
+        end_time_usecs (int, optional): The end time for the recoveries (in
+        microseconds). If None, this value will not be included in the request.
 
     Returns:
         A dictionary representing the JSON response from the API containing
         information about the recoveries.
     """
 
     # Define the URL to send the request to
-    url = 'https://helios.cohesity.com/v2/data-protect/recoveries'
+    url = (
+        f"https://helios.cohesity.com/v2/data-protect/recoveries?&"
+        "includeTenants=true"
```

Comment:
Can this be removed as it's already present in params?

Reply:
done